core/templates/dev/head/player/ConversationSkinDirective.js

```diff
@@ -706,6 +706,10 @@ oppia.directive('conversationSkin', [function() {
           return $scope.windowWidth < TWO_CARD_THRESHOLD_PX;
         };
 
+        $scope.isWindowTall = function() { 
+          return document.body.scrollHeight > window.innerHeight;
```

Comment:
OK, no worries. I seem to remember that there are some differences between `$document` and `document`, so let's just use the one that does work.